src/go/rpk/pkg/cli/cmd/cluster/license/info.go

```diff
@@ -53,8 +53,9 @@ func newInfoCommand(fs afero.Fs) *cobra.Command {
 						Organization string
 						Type         string
 						Expires      string
-						Expired      bool `json:"license_expired,omitempty"`
-					}{info.Properties.Organization, info.Properties.Type, tm, expired}, "", "  ")
+						Checksum     string `json:"ChecksumSHA256,omitempty"`
```

Comment:
@graphcareful pointed out the casing inconsistency to me in this response -- I think we should switch all of these to lower snake case,
```go
						Organization string `json:"organization"`
						Type         string `json:"type"`
						Expires      string `json:"expires"`
						Checksum     string `json:"checksum_sha256,omitempty"`
						Expired      bool   `json:"license_expired,omitempty"`
					}{info.Properties.Organization, info.Properties.Type, tm, info.Properties.Checksum, expired}, "", "  ")
```